### DISCUSSION TITLE
Can't execute the Job

### DISCUSSION THREAD

**Discussion by lopermo** [quirrel-dev/quirrel#470] [Q&A]
Hi!

I'm trying to write an example with Next in stackblitz for an article I'm writing about Quirrel, but I can't get  the job to be fully executed.

I get the message of "Created Job", and then after the delay, "Executing Job", but it never executes itself.

https://stackblitz.com/edit/quirrel-test?file=pages/index.js

**Comment by Skn0tt** [comment]
Wow, this Stackblitz thing is impressive!

Quirrel works by making Webhook-like API calls, and it seems like Stackblitz doesn't yet fully support those. The working one of the two buttons accesses Quirrel from the frontend, which is generally not a good idea for data security reasons.
The second button doesn't work because it calls Quirrel from an API Route.
I'd suspect Stackblitz correctly rewrites client-side HTTP requests, but not HTTP requests made from a Next.js API Route. @garrisons can you give more context on this?
